terraform/Makefile

```
@@ -43,8 +43,12 @@ import_resources: rename_resources
 plan: import_resources
 	terraform plan
 
+.PHONY: delete_lambda_versions
+delete_lambda_versions: import_resources
```

```suggestion
delete_lambda_versions: import_resources check_python 
```

Same for any other recipe that invokes `python`.